addons/l10n_es/data/account_tax_data.xml

```diff
@@ -773,14 +774,24 @@
     <record id="account_tax_template_s_iva0" model="account.tax.template">
         <field name="description">S_IVA0</field>
         <field name="type_tax_use">sale</field>
-        <field name="name">IVA Exento Repercutido</field>
+        <field name="name">IVA Exento Repercutido Sujeto</field>
         <field name="account_id" ref="l10n_es.account_common_472"/>
         <field name="refund_account_id" ref="l10n_es.account_common_472"/>
         <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         <field name="amount" eval="0"/>
         <field name="amount_type">percent</field>
         <field name="tax_group_id" ref="tax_group_iva_0"/>
     </record>
+    <record id="account_tax_template_s_iva0_ns" model="account.tax.template">
+        <field name="description">S_IVA0_NS</field>
+        <field name="type_tax_use">sale</field>
+        <field name="name">IVA Exento Repercutido No Sujeto</field>
+        <field name="account_id" ref="l10n_es.account_common_472"/>
+        <field name="refund_account_id" ref="l10n_es.account_common_472"/>
+        <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
+        <field name="amount" eval="0"/>
+        <field name="amount_type">percent</field>
+        <field name="tax_group_id" ref="tax_group_iva_0"/>
```

Comment:
The module won't be installable. You're missing a `</record>` 
Always test your code ;)